tools/internal_ci/macos/grpc_cfstream.cfg

```diff
@@ -16,4 +16,8 @@
 
 # Location of the continuous shell script in repository.
 build_file: "grpc/tools/internal_ci/macos/grpc_run_bazel_tests.sh"
+env_vars {
+  key: "RUN_TESTS_FLAGS"
+  value: ""
```

Comment:
Do we have to add this env var even if it's empty?

Reply:
It's not necessary, I'll remove it.